EIPS/eip-1521.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: 1521
+title: Standard for Insurance Policies as ERC-721 Non Fungible Tokens
+author: Christoph Mussenbrock <@christoph2806>
```

Comment:
Need to use parenthesis instead of angle-brackets for github usernames.

Reply:
ok I'll modify accordingly